File: examples/java/src/test/java/dev/selenium/getting_started/InstallDriversTest.java
package dev.selenium.getting_started;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.edge.EdgeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.ie.InternetExplorerDriver;

import io.github.bonigarcia.wdm.WebDriverManager;

public class Third {  
  
    public static void main(String[] args) {  
      
           // System Property for Chrome Driver   
        System.setProperty("webdriver.chrome.driver", "D:\\ChromeDriver\\chromedriver.exe");  
          
             // Instantiate a ChromeDriver class.     
        WebDriver driver=new ChromeDriver();  
          
           // Launch Website  
        driver.navigate().to("http://www.javatpoint.com/");  
          
         //Maximize the browser  
          driver.manage().window().maximize();  
          
          //Scroll down the webpage by 5000 pixels  
        JavascriptExecutor js = (JavascriptExecutor)driver;  
        js.executeScript("scrollBy(0, 5000)");   
          
         // Click on the Search button  
        driver.findElement(By.linkText("Core Java")).click();     
  
    }  
  
}  

File: examples/kotlin/src/test/kotlin/dev/selenium/getting_started/InstallDriversTest.kt
package dev.selenium.getting_started

import io.github.bonigarcia.wdm.WebDriverManager
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import org.openqa.selenium.WebDriver
import org.openqa.selenium.chrome.ChromeDriver
import org.openqa.selenium.edge.EdgeDriver
import org.openqa.selenium.firefox.FirefoxDriver
import org.openqa.selenium.ie.InternetExplorerDriver


class InstallDriversTest {
    @Test
    @Disabled("Do not run in CI")
    fun chromeSession() {
        WebDriverManager.chromedriver().setup()
        val driver: WebDriver = ChromeDriver()
        driver.quit()
    }

    @Test
    @Disabled("Do not run in CI")
    fun edgeSession() {
        WebDriverManager.edgedriver().setup()
        val driver: WebDriver = EdgeDriver()
        driver.quit()
    }

    @Test
    @Disabled("Do not run in CI")
    fun firefoxSession() {
        WebDriverManager.firefoxdriver().setup()
        val driver: WebDriver = FirefoxDriver()
        driver.quit()
    }

    @Test
    @Disabled("Do not run in CI")
    fun ieSession() {
        WebDriverManager.iedriver().setup()
        val driver: WebDriver = InternetExplorerDriver()
        driver.quit()
    }
}